migration/src/lib.rs

```diff
@@ -16,6 +16,7 @@ mod m20230203_205959_improve_upsert_perf;
 mod m20230224_093722_performance_improvements;
 mod m20230310_162227_add_indexes_to_bg;
 mod m20230317_121944_remove_indexes_for_perf;
+mod m20230510_183736_add_indices_to_assets;
```

Comment:
This migration was included with https://github.com/metaplex-foundation/digital-asset-rpc-infrastructure/pull/110